### ISSUE TITLE
What dose 'angle' mean in yolov3.cfg?

### ISSUE THREAD

**Issue by sogou-li** [pjreddie/darknet#967]
If I set angle=30, it means:
 i ) : rotate image randomly within 0 to 30 degree 
 ii ) : rotate image randomly within -30 to 30 degree 

which is right? Or both are wrong?

**Comment by developer0hye** [comment]
did you solve the your question?

**Comment by sogou-li** [comment]
I think so. By reading the source code, I thought the second is right.

**Comment by developer0hye** [comment]
@sogou-li 
thank you

**Comment by michalgdak** [comment]
Reading https://groups.google.com/forum/#!topic/darknet/0-u3Tl4RGhg I guess it's radians and is not being taken into account while traning Yolo

**Comment by pachecoberlin** [comment]
> I think so. By reading the source code, I thought the second is right.

This is correct as you can see in code in random_augment_args in image.c [ https://github.com/pjreddie/darknet/blob/master/src/image.c#L1183](url)

**Comment by yangyangHu** [comment]
source code in darknet/src/image.c
augment_args random_augment_args(image im, float angle, float aspect, int low, int high, int w, int h)
{
    ....
    float rad = rand_uniform(-angle, angle) * TWO_PI / 360.; #random rotate in [-angle, angle]
    ....
    a.rad = rad;
    ....
    return a;
}

**Comment by mhe-wyze** [comment]
How about the bboxes in the image, are they automatically rotating to the correct location?

**Comment by mhe-wyze** [comment]
@sogou-li @developer0hye @michalgdak @pachecoberlin @yangyangHu
